### ISSUE TITLE
show review popup

### ISSUE THREAD

**Issue by younesyzd** [pushandplay/cordova-plugin-apprate#186]
Hello 
i need to show review popup every time user connect to game and when the user review my game stop to show the review popup ? that is possible in this plugin ?

**Comment by ldinatale1182** [comment]
Yes, you have complete control as to when you show the review popup.
